content/docs/services/elasticsearch17.md

```diff
@@ -4,7 +4,7 @@ menu:
     parent: services
 title: Elasticsearch 1.7
 name: "elasticsearch17"
-description: "[Elasticsearch](https://www.elastic.co/) version 1.7"
+description: "[Elasticsearch](https://www.elastic.co/products/elasticsearch) version 1.7 a distributed, RESTful search and analytics engine"
```

Comment:
BUT, these descriptions are actually for the page metadata (for things like url previews on slack) and are not converted to HTML so should not include links. I'll update that on all the pages.